sdk/python/tests/unit/permissions/auth/server/test_utils.py

```diff
@@ -0,0 +1,61 @@
+import assertpy
+import pytest
+
+from feast import Entity, FeatureView, OnDemandFeatureView, StreamFeatureView
+from feast.permissions.action import AuthzedAction
+from feast.permissions.permission import Permission
+from feast.permissions.policy import RoleBasedPolicy
+from feast.permissions.server.utils import AuthManagerType, str_to_auth_manager_type
+
```

Comment:
Only tests describe?

Reply:
yes, this class tests the functionality written in [utils.py](https://github.com/RHEcosystemAppEng/feast/blob/feast-rbac/sdk/python/feast/permissions/server/utils.py) file. and also some common code related to auth tests.